packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -4,22 +4,27 @@ import {
   accountSelectors,
   cacheUsersSelectors,
   hlsUtils,
-  Genre,
   playerSelectors,
   playerActions,
   queueActions,
   queueSelectors,
   reachabilitySelectors,
   RepeatMode,
   FeatureFlags,
-  encodeHashId
+  encodeHashId,
+  Genre
 } from '@audius/common'
-import { Platform, StyleSheet, View } from 'react-native'
-import MusicControl from 'react-native-music-control'
-import { Command } from 'react-native-music-control/lib/types'
-import type { OnProgressData } from 'react-native-video'
-import Video from 'react-native-video'
+import TrackPlayer, {
+  AppKilledPlaybackBehavior,
+  Capability,
+  Event,
+  State,
+  usePlaybackState,
```

Comment:
Were you able to confirm these changes fix the android bug as well?

Reply:
About to try that